website/docs/language/settings/backends/s3.mdx

```diff
@@ -314,8 +314,8 @@ provider "aws" {
   # No credentials explicitly set here because they come from either the
   # environment or the global credentials file.
 
-  assume_role = {
-    role_arn = "${var.workspace_iam_roles[terraform.workspace]}"
+assume_role {
```

Comment:
```suggestion
  assume_role {
```
I think we need to indent this field in the example because it's a part of the `provider` block.